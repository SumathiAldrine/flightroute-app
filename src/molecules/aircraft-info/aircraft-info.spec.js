import React from 'react';
import renderer from 'react-test-renderer';
import AircraftInfo from './index';

test('renders Aircraft Information', () => {
  const tree = renderer.create(<AircraftInfo />).toJSON();
  
  expect(tree).toMatchSnapshot();
});
