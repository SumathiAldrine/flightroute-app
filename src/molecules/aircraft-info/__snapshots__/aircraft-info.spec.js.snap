// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Aircraft  1`] = `
<div
  className="aircraft-info__aircraft"
>
  <div
    className="aircraft-info__tail-container"
  >
    <img
      alt="Tail"
      className="aircraft-info__flight-tail"
      src="tail_icon.png"
    />
    <span
      className="aircraft-info__tailNo"
      tabIndex="0"
    />
  </div>
  <div
    className="aircraft-info__progress-container"
  >
    <div
      className="aircraft-info__progress-line"
    >
      <img
        alt="Airline"
        className="aircraft-info__progress-airplane"
        src="flight_icon.png"
      />
      <div
        className="aircraft-info__progress-line--complete"
      />
      <div
        className="aircraft-info__progress-point aircraft-info__progress-point--finish"
      />
    </div>
  </div>
  <div
    className="aircraft-info__duration"
    tabIndex="0"
  />
</div>
`;

exports[`renders Aircraft Information 1`] = `
<div
  className="aircraft-info__aircraft"
>
  <div
    className="aircraft-info__tail-container"
  >
    <img
      alt="Tail"
      className="aircraft-info__flight-tail"
      src="tail_icon.png"
    />
    <span
      className="aircraft-info__tailNo"
      tabIndex="0"
    />
  </div>
  <div
    className="aircraft-info__progress-container"
  >
    <div
      className="aircraft-info__progress-line"
    >
      <img
        alt="Airline"
        className="aircraft-info__progress-airplane"
        src="flight_icon.png"
      />
      <div
        className="aircraft-info__progress-line--complete"
      />
      <div
        className="aircraft-info__progress-point aircraft-info__progress-point--finish"
      />
    </div>
  </div>
  <div
    className="aircraft-info__duration"
    tabIndex="0"
  />
</div>
`;

exports[`renders learn react link 1`] = `
<div
  className="aircraft-info__aircraft"
>
  <div
    className="aircraft-info__tail-container"
  >
    <img
      alt="Tail"
      className="aircraft-info__flight-tail"
      src="tail_icon.png"
    />
    <span
      className="aircraft-info__tailNo"
      tabIndex="0"
    />
  </div>
  <div
    className="aircraft-info__progress-container"
  >
    <div
      className="aircraft-info__progress-line"
    >
      <img
        alt="Airline"
        className="aircraft-info__progress-airplane"
        src="flight_icon.png"
      />
      <div
        className="aircraft-info__progress-line--complete"
      />
      <div
        className="aircraft-info__progress-point aircraft-info__progress-point--finish"
      />
    </div>
  </div>
  <div
    className="aircraft-info__duration"
    tabIndex="0"
  />
</div>
`;
